airbyte-api/src/main/openapi/config.yaml

```diff
@@ -906,6 +906,114 @@ paths:
           description: Connection not found
         "422":
           $ref: "#/components/responses/InvalidInput"
+  /v1/operations/create:
```

Comment:
Why do you prefer to have separate endpoints for operations as opposed to just have it as part of the sync endpoints? Not saying it's wrong / bad. Just curious how you are thinking about it.

Reply:
This way gives more flexibility if one day, we need/want to re-use and share operations across connections/sync.

Even if the UI would always tie syncs to operations, the API could let dev-users do differently if they wished.